float64 thrust_des
float64 phi_des
float64 theta_des
float64 r_des
